### PR TITLE
fix(aio): remove title from callout

### PR DESCRIPTION
**Pull request by sjtrimble**
Per discussion - removed title from callout to remain consistent style-wise.

## PR Checklist
Does please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

```
[x] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[ ] Documentation content changes
[ ] angular.io application / infrastructure changes
[ ] Other... Please describe:
```

## Does this PR introduce a breaking change?
```
[ ] Yes
[x] No
```

### CODE REVIEW

**Comment by sjtrimble** [comment]
@gkalpak Do you think we could add tests for this - to prevent titles (headers) in callouts and alerts?

**Comment by petebacondarwin** [comment]
@sjtrimble we could add an HTML post-processor to the doc-gen to check for this.

File: aio/content/guide/ts-to-js.md [hunk 1]
```diff
@@ -247,20 +247,18 @@ next to the original _ES5_ version for comparison:
...
 
-<div class="callout is-helpful">
-
-  {@a name-constructor}
+{@a name-constructor}
 
-  ###  Name the constructor
+<div class="callout is-helpful">
 
+  {@a getters-setters}
```
Comment (petebacondarwin) [inline aio/content/guide/ts-to-js.md]: why move this anchor?

Comment (sjtrimble) [inline aio/content/guide/ts-to-js.md]: It affected the spacing -- styles weren't fully applying. This was the only way I was able to get it to look as intended.

Comment (petebacondarwin) [inline aio/content/guide/ts-to-js.md]: To be honest I don't know if we even need this anchor. The "Properties with getters and setters" will have its own anchor auto-attached. I can't find anywhere in the docs that links to `#getters-setters`.

Comment (sjtrimble) [inline aio/content/guide/ts-to-js.md]: I can remove ... was tempted to but wasn't quite sure as I'm not usually in the `.md` files

Comment (petebacondarwin) [inline aio/content/guide/ts-to-js.md]: I moved the anchor back to the original place and it didn't appear to have any visual difference on Chrome, Safari or Firefox (all on OS/X). Are you sure this needs to move?

**Review by petebacondarwin — CHANGES_REQUESTED** [review]

**Review by petebacondarwin — APPROVED** [review]

**Comment by petebacondarwin** [comment]
screen shot 2017-07-14 at 12 34 43

**Review by Foxandxss — APPROVED** [review]

**Comment by petebacondarwin** [comment]
Thanks @Foxandxss

**Pull request merged**